pkg/apiserver/apiserver.go

```diff
@@ -35,7 +36,10 @@ import (
 	"github.com/spf13/afero"
 )
 
-const maxRequestBodyBytes = 1024 * 1024 * 1024
+const (
+	maxRequestBodyBytes = 1024 * 1024 * 1024
+	Root                = "/data"
```

Comment:
suggestion: this default is related only to the filesystem storage, we should declare it there

I think it can confusing when you see this code used as `apiserver.Root` somewhere else. Is it a hostname for the APIServer? Which root is it exactly? How is this tied to APIServer?

I think we should tie the filesystem storage root to the filesystem implementation. So this constant would be declared inside the filesystem storage package, and can then be imported with the `NewStorageImpl` constructor function.

For example:

```go
package file

const (
    DefaultStorageRoot = "/data"
)

// ...
```

```go
package apiserver

import "file"

// ...
func (c completedConfig) New() (*WardleServer, error) {
    // ...
    storageImpl := NewStorageImpl(afero.NewOsFs(), file.DefaultStorageRoot)
    // ...
}
```

This way we would keep APIServer concerns separate from the backing storage concerns.

Reply:
makes sense